pkg/console/subresource/configmap/configmap.go

```diff
@@ -155,6 +159,17 @@ func servingInfo() yaml.MapSlice {
 	}
 }
 
+func consoleProviders(providers operatorv1.ConsoleProviders) yaml.MapSlice {
+	if providers.Statuspage != nil {
```

Comment:
This should be good for now. I imagine eventually we will need to `slice := yaml.MapSlice{ }`, and then conditionally append items one-by-one, but not necessary until we have other providers.